src/list_of_files.cmake

```diff
@@ -0,0 +1,71 @@
+set(fortran_src
+  ${CMAKE_CURRENT_SOURCE_DIR}/addfield.f
```

Comment:
`${CMAKE_CURRENT_SOURCE_DIR}` is not necessary.
Please remove.

Reply:
Ah yes, I did this before I started removing them